### PR TITLE
Alphabetize Generator Entries

### PR DESCRIPTION
**Pull request by LindseySaari**
## Description of change
When the vets-api module generator runs, it inserts entries for the newly generated module/engine into the simplecov_helper spec_helper, Gemfile and routes files using the thor insert_into_file method. The current method of insertion doesn't alphabetize the entries and just inserts after an existing file comment. This PR inserts the new entries with respect to alphabetization. A helper/module was created to dry up the repeated parts of the insertion code.

Additionally, the commit_message_methods array/tests were removed because we no longer force a commit message when invoking the generator. We were previously using a specific commit message for analytical tracking purposes, but we are now taking a new approach using DataDog. 

## Original issue(s)
department-of-veterans-affairs/va.gov-team#19242

## Things to know about this PR
- [x] Specs passing locally
- [x] Tested locally
- [x] Current entries were not in alphabetical order/alphabetized current entries

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -46,24 +46,24 @@
...
     add_group 'ClaimsApi', 'modules/claims_api/'
     add_group 'CovidVaccine', 'modules/covid_vaccine/'
+    add_group 'HealthQuest', 'modules/health_quest/'
+    add_group 'Mobile', 'modules/mobile/'
     add_group 'OpenidAuth', 'modules/openid_auth/'
+    add_group 'Policies', 'app/policies'
+    add_group 'Serializers', 'app/serializers'
+    add_group 'Services', 'app/services'
+    add_group 'Swagger', 'app/swagger'
+    add_group 'Uploaders', 'app/uploaders'
```
Comment (thilton-oddball) [inline spec/spec_helper.rb:60]: Do you want Uploaders to come after TestUserDashboard?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -46,24 +46,24 @@
...
     add_group 'TestUserDashboard', 'modules/test_user_dashboard/'
     add_group 'VBADocuments', 'modules/vba_documents/'
     add_group 'Veteran', 'modules/veteran/'
     add_group 'VeteranVerification', 'modules/veteran_verification/'
-    add_group 'OpenidAuth', 'modules/openid_auth/'
     add_group 'VAOS', 'modules/vaos/'
```
Comment (thilton-oddball) [inline spec/spec_helper.rb:65]: Should VAOS come before VBADocuments?

**Review by thilton-oddball — APPROVED** [review]
Just a couple of small items to get in alphabetical order but otherwise looks good!

File: spec/spec_helper.rb (hunk 1 above)
Comment (LindseySaari) [reply spec/spec_helper.rb:60]: Wow, I clearly don't know the English alphabet lol. I had a merge conflict and must have missed the typos

**Pull request merged**